conjure-lib/src/main/java/com/palantir/conjure/java/lib/SafeLong.java

```diff
@@ -56,4 +56,8 @@ public final String toString() {
         return Long.toString(longValue());
     }
 
+    @Override
+    public final int compareTo(SafeLong other) {
+        return Long.compare(longValue(), other.longValue());
```

Comment:
what if other is null? do we want some special handling?

Reply:
So I think this is actually pretty unlikely due to how we've really tried to purge nulls from our codebases, and also seems OK according to the javadoc of Comparable:

```
     * @throws NullPointerException if the specified object is null
```

Reply:
yep - I don't deal much with nulls, but my general understanding is that special null handling is useful only when it makes a tricky error easier (e.g. if the NPE happens 10 lines down, in a different method). Here you're getting an NPE on line 1 anyway.